web/controllers/rules/join/join_rules.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Dwylbot.Rules.Join do
+  @moduledoc """
+  join rules together when necesssary:
+  - in-progress label added: combine no_assignee with no_estimation
+  - multiple reviewers added: combine add assignees rule
```

Comment:
I'll implement this (combine reviewers errors) on another PR, this one is full I think